CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.8.1 / 2023-04-27
+
+* [BUGFIX] Update OkHttp version to 4.11.0. See [#161](https://github.com/DataDog/dd-sdk-android-gradle-plugin/pull/161)
```

Comment:
no, Android SDK is on the 3.x branch still, because of the API 19 support. Anyway, this bug happens only if request is rejected by the server (and I guess for multipart form request, which is not used by Android SDK).

Reply:
we are using multipart for SR though now :)

Reply:
Well, the fix was backported from 5.x branch to 4.x branch only. We will switch to 4.x eventually once we cut off API 19 support in SDK v2.